job-service-caller/pom.xml

```diff
@@ -60,7 +60,7 @@
                     </execution>
                 </executions>
                 <configuration>
-                    <autoPull>always</autoPull>
+                    <autoPull>on</autoPull>
```

Comment:
Remove the changes which are unrelated to this Jira, such as this one.  These can be pushed in a separate commit.  We need to keep commits focused.  See https://github.com/salemove/github-review-helper/blob/master/doc/rules.md#focused.